ios/RNSScreenStackHeaderConfig.m

```diff
@@ -355,6 +355,21 @@ + (void)updateViewController:(UIViewController *)vc withConfig:(RNSScreenStackHe
   if (shouldHide) {
     return;
   }
+  
+  // if we don't explicitly change the direction in config, we should get it from the phone settings
+  if (config.direction == UISemanticContentAttributeUnspecified) {
+    if ([NSLocale preferredLanguages].count != 0 &&
+         [NSLocale characterDirectionForLanguage:[[NSLocale preferredLanguages] objectAtIndex:0]] == NSLocaleLanguageDirectionRightToLeft) {
+       navctr.view.semanticContentAttribute = UISemanticContentAttributeForceRightToLeft;
+       navctr.navigationBar.semanticContentAttribute = UISemanticContentAttributeForceRightToLeft;
+     } else {
+       navctr.view.semanticContentAttribute = UISemanticContentAttributeForceLeftToRight;
+       navctr.navigationBar.semanticContentAttribute = UISemanticContentAttributeForceLeftToRight;
+     }
+  } else {
+    navctr.view.semanticContentAttribute = config.direction;
```

Comment:
is it necessary to set it on the navigation view? Setting on the navbar isn't enough?

Reply:
Setting it only on the navbar does not change the side from which we can dismiss the screen with swipe gesture.